common/rabbitmq/templates/etc/_rabbitmq-custom-config.tpl

```diff
@@ -0,0 +1,15 @@
+{{ if .Values.customConfig }}
+## RabbitMQ custom config
+## Generated by helm template etc/_rabbitmq-custom-config.tpl
+## For more information please find helm chart
+##     https://github.com/sapcc/helm-charts/tree/master/common/rabbitmq
+
+## For more information about RabbitMQ configuration please find
+##     https://www.rabbitmq.com/configure.html
+
+{{- range $k, $v := .Values.customConfig }}
+{{ if not (eq $v nil) }}
```

Comment:
Personally, I am more a fan of using everywhere 
``{{- `` and only in the places you really want a newline to leave the `-` away.
That serves two purposes:
1. It is consistent and easily to remember
2. You can indent the loops and conditions logically.
I.e.
```
{{- range $k, $v := .Values.customConfig }}
  {{- if kindIs "invalid" $value }}
{{ $k }} = {{ $v }}
  {{- end }}
{{- end }}

No a requirement from me to change, but just a suggestion.

```

Reply:
Th `kindIs` needs a "not", right?

Reply:
Done!